core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -39,7 +39,7 @@ object Automation {
             gpp = specialistInfo.greatPersonPoints.sumValues().toFloat()
         }
         gpp = gpp * (100 + city.currentGPPBonus) / 100
-        rank += gpp * 3 // GPP weight
+        rank += gpp * 1 // GPP weight
```

Comment:
What exactly is the reasoning for this? The Automation doesn't use specialists very often in the first place.

Reply:
The AI currently doesn't distinguish between types of great person points, and doesn't know well how to use the ones they produce, which makes a calculation like this not very useful. I think it's best to set it to a low value, to act as a 'tiebreaker' between an engineer slot and an unimproved hill.